fastlane/lanes/release.rb

```diff
@@ -121,6 +121,16 @@
     create_release_management_pull_request('trunk', "Merge #{new_version} code freeze into trunk")
   end
 
+  lane :test_git_push do |options|
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-Android/blob/cf1c232da43c12464ca9d5a516ae263f286244c2/fastlane/lanes/release.rb#L124"></span>Lint/UnusedBlockArgument: Unused block argument - <code>options</code>. You can omit the argument if you don't care about it.</td>
    </tr>
  </tbody>
</table>